src/app/(dashboard)/projects/[projectId]/payments/paymentCreateModal.tsx

```diff
@@ -1,25 +1,29 @@
+import {
+  PaymentAddressDto,
+  PaymentRecipientDto,
+} from "@/app/api/payments/route";
 import { PaymentSplitDto } from "@/app/api/payments/service/paymentSplitsService";
 import { Button } from "@/components/ui/button";
 import { Dialog, DialogContent, DialogTrigger } from "@/components/ui/dialog";
-import { useSplitsClient } from "@0xsplits/splits-sdk-react";
-import { useAccount } from "wagmi";
+import { useCreateSplit, useSplitsClient } from "@0xsplits/splits-sdk-react";
+import axios from "axios";
+import { useAccount, useChainId } from "wagmi";
 
 type CreatePaymentAddressModalProps = {
   projectId: string;
   paymentSplits: PaymentSplitDto[];
+  onCreate: (param: PaymentAddressDto) => void;
 };
 
 export function CreatePaymentAddressModal({
   projectId,
   paymentSplits,
+  onCreate,
 }: CreatePaymentAddressModalProps) {
   const account = useAccount();
+  const chainId = useChainId();
 
-  const splitsClient = useSplitsClient({
-    // TODO: Use connected chainId
-    chainId: 8453,
-    publicClient: window.ethereum!,
-  });
+  const { createSplit, status, txHash, error } = useCreateSplit();
```

Comment:
is this actually being used on this page?

Reply:
Yes, updated code pushed